selfdrive/locationd/paramsd.py

```diff
@@ -224,13 +224,13 @@ def main():
       liveParameters.roll = roll
       liveParameters.angleOffsetAverageDeg = angle_offset_average
       liveParameters.angleOffsetDeg = angle_offset
+      liveParameters.angleOffsetValid = all((avg_offset_valid, total_offset_valid))
```

Comment:
should we add a valid for each parameter? can refactor into a nice struct in cereal

Reply:
Not really I think. The other ones are rare enough that rerunning is necessary to see what is happening. It will create more fields without much varying information.